high_card.rb

```diff
@@ -1,5 +1,48 @@
+require "pry"
+
 # Use these two arrays to generate a deck of cards.
 ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+deck_cards = []
+
+# class Cards
+
+  def deck
+    # It looks like we will need a nested for loop
+    for rank in ranks do #we used an each loop in the hw review; "each is far more ubiquitous in Ruby"
```

Comment:
If you're still not comfortable with `.each`, strongly encourage you to try it out again here in place of `for in`.